Topics/User_Experience/User_Research.md

```diff
@@ -0,0 +1,102 @@
+## Resources for User Research domain
+
+### Overview
+
+User research is an essential part of the user experience (UX) design process. It involves gathering information about users and their needs, behaviors, and attitudes through various research methods. The goal of user research is to create a user-centered design by understanding the needs and goals of the users and incorporating this information into the design process. the key topics in user research include User Personas, User Testing, Surveys and Questionnaires, Interviews, and Contextual Inquiry.
+
+User research can be conducted through various methods, such as surveys, interviews, focus groups, usability testing, and ethnographic studies. These methods allow researchers to gather information about users' experiences, behaviors, and preferences, which can then be used to inform design decisions.
+
+### Key Topics in User Research
+
+1. User Personas: User personas are fictional representations of your target users. They help you understand the needs, goals, and behaviors of your users, which can inform the design of your product. Creating user personas involves conducting research to gather information about your users, such as demographic data, behavioral patterns, and attitudes.
+
+Example: I want to study space finder application, I would create a user persona first for the UX
+
+Step 1: Define a user: our user is likely to be a student or working professional who is looking for a quiet, comfortable, and productive space to study or work.
```

Comment:
Similarly, each steps below are under the Example, so indenting it can make it easier to read.